docs/nydusd.md

```diff
@@ -204,7 +206,32 @@ Document located at: https://github.com/adamqqqplay/nydus-localdisk/blob/master/
         "endpoint": "region.aliyuncs.com",
         "access_key_id": "",
         "access_key_secret": "",
-        "bucket_name": ""
+        "bucket_name": "",
+        "object_prefix": "nydus/"
```

Comment:
We don't have a default OSS object prefix. Why set it here as an example? If users follow nydusify.md to create new images, they won't be able to access it with the example backend config here.

Reply:
Okay, let's also update `nydusify.md`, it shows how to put blobs into a directory, tells the user the need of tail `/`.

Reply:
Updated.